mpm/jpqr/id.go

```diff
@@ -73,3 +73,13 @@ func ParseID(c *mpm.Code) (*ID, error) {
 	}
 	return nil, errors.New("missing JPQR-ID")
 }
+
+// ParseIDFromString validates and parses given string as JPQR-ID.
+func ParseIDFromString(v string) (*ID, error) {
+	return ParseID(&mpm.Code{
+		MerchantAccountInformation: []tlv.TLV{
+			// We have not checked the lengths of the tag and the length of tlv fields strictly for now so set dummy value "xx"
+			{Tag: "xx", Length: "xx", Value: v},
```

Comment:
[IMO] As far as we know, the length of the valid JPQR-ID is 68. So, we can set 68 as length.

Reply:
Should we make sure that the length of `c.MerchantAccountInformation` is exactly 74 (the length of JPQR-ID including TLV) in `ParseID`?

Reply:
@ymt2 It is hard to understand I fixed to the correct values.